### PR TITLE
fix: updated footer, updated footer data

### PR DESCRIPTION
**Pull request by Ebmili**
## Jira issue

JIRA issue: [FSU-XX](https://klau.atlassian.net/browse/FSU-XX)

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/blocks/footer/footerData.js [hunk 1]
```diff
@@ -102,27 +74,11 @@ const footerData = {
...
-      title: "Adopcja za granicę",
-      link: "",
-    },
-    {
-      title: "Znalazły nowy dom",
-      link: "",
-    },
-    {
-      title: "Wsparcie po adopcji",
+      title: "Zasady adopcjii",
```
Comment (AmandaKr) [inline packages/design-system/blocks/footer/footerData.js:77]:
```suggestion
      title: "Zasady adopcji",
```

**Comment by AmandaKr** [comment]
I think it would be a good idea to include link from jira(that's what the PR template was created for) and a short description of the task :)

**Comment by im-klau** [comment]
@Ebmili  This PR's title doesn't match the naming convention that we follow. You used "ci" which means that changes are related to continuous integration, but that's not true :D 

take a look at the possible types 👇:

Zrzut ekranu 2023-02-27 o 10 59 35

I guess this one could be `refactor: ...` or maybe `fix: ...` 🤔

File: packages/design-system/blocks/footer/footerData.js (hunk 1 above)
Comment (Ebmili) [reply packages/design-system/blocks/footer/footerData.js]: Done ;)

**Comment by im-klau** [comment]
> @Ebmili This PR's title doesn't match the naming convention that we follow. You used "ci" which means that changes are related to continuous integration, but that's not true :D
> 
> take a look at the possible types 👇:
> 
> Zrzut ekranu 2023-02-27 o 10 59 35
> 
> I guess this one could be `refactor: ...` or maybe `fix: ...` 🤔


@Ebmili take a look at my previous message, pls 🙏 it's important to have consistent PR names

**Review by im-klau — APPROVED** [review]

**Review by AmandaKr — APPROVED** [review]

**Pull request merged**